src/app/@theme/theme.module.ts

```diff
@@ -28,6 +28,7 @@ import { DEFAULT_THEME } from './styles/theme.default';
 import { DARK_THEME } from './styles/theme.dark';
 
 import { FormsModule } from '@angular/forms';
+import { JobActionsMenuComponent } from './components/job-actions-menu/job-actions-menu.component';
```

Comment:
The `job-actions-menu` folder/component should be placed inside of `/src/app/pages/dashboard/job-result/` folder i.e. the dashboard module, not the themes module.